### PR TITLE
Update docs to match the new triague guidelines

### PR DESCRIPTION
**Pull request by stsewd**
Refs to https://github.com/rtfd/readthedocs.org/issues/2673#issuecomment-398149984 and https://github.com/rtfd/readthedocs.org/pull/4180/

### CODE REVIEW

File: docs/contribute.rst [hunk 1]
```diff
@@ -111,8 +111,7 @@ label. The reported issue …
     in the past or it was decided that the proposed feature will not be
     implemented because it does not conform with the overall goal of Read the
     Docs. Also if you happen to know that the problem was already reported,
-    label the ticket with **Status: duplicate**, reference the other ticket
```
Comment (stsewd) [inline docs/contribute.rst]: We don't have the duplicate label anymore

File: docs/contribute.rst [hunk 2]
```diff
@@ -140,18 +139,16 @@ label. The reported issue …
     - *My builds stopped working. Please help!*
       Ask for a link to the build log and for which project is affected.
 
-… is a valid enhancement proposal
-    If the ticket contains an enhancement proposal that aligns with the goals
-    of Read the Docs, then add the label **Enhancement**. If the proposal
+… is a valid feature proposal
```
Comment (stsewd) [inline docs/contribute.rst]: I think `enhancement` is now `feature`, but we also have `Improvement` (that's why I removed some examples here)

File: docs/contribute.rst [hunk 3]
```diff
@@ -208,14 +205,6 @@ few more at hand to further categorize issues.
     If the issue is urgent, assign this label. In the best case also go forward to
     resolve the ticket yourself as soon as possible.
 
-*Community Effort*
```
Comment (RichardLitt) [inline docs/contribute.rst]: We still have this label. :/ The other PR doesn't actually refer to removing it, it just removed it (unless I am missing something?)

Comment (stsewd) [reply docs/contribute.rst]: Well, here is a little of discussion https://github.com/rtfd/readthedocs.org/pull/4180#discussion_r193112104 and the commit https://github.com/rtfd/readthedocs.org/pull/4180/commits/d98189254c383c3f8de17ed0768581c8877df35f

Comment (RichardLitt) [reply docs/contribute.rst]: Yeah, I saw that discussion and the commit; it just doesn't seem like dropping the 'Community Effort' label was discussed, it was just included in the commit.

Comment (agjohnson) [reply docs/contribute.rst]: I've already started retriaging these issues, but yes, there are still some left. I'm +1 on just removing from the docs for now.

**Comment by humitos** [comment]
> Do we want to add mention of Improvement here as well?

@stsewd I think this is the only missing piece to merge this PR. Want to write a paragraph for this?

**Comment by stsewd** [comment]
@humitos I'll take a look tomorrow morning

**Comment by stsewd** [comment]
Done!

**Comment by agjohnson** [comment]
Thanks for putting this together!

**Pull request merged**